app/utils/__tests__/gameLogic.test.ts

```diff
@@ -1,7 +1,24 @@
 import { getInitialGameState } from '../gameLogic';
 import { GameState } from '../../types/game';
+import * as deckUtils from '../deckUtils';
+
+jest.mock('../deckUtils', () => ({
+  ...jest.requireActual('../deckUtils'),
```

Comment:
I found this way to do not mock this function, while tests still run. Are you ok with this?